pkgs/development/python-modules/lunarcalendar/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+
+, python-dateutil
+, ephem
+, pytz
+
+, pytestCheckHook
+}:
+buildPythonPackage rec {
+  pname = "LunarCalendar";
+  version = "0.0.9";
+
+  disabled = pythonOlder "2.7";
+
+  src = fetchFromGitHub {
+    owner = "wolfhong";
+    repo = "LunarCalendar";
+    rev = "885418ea1a2a90b7e0bbe758919af9987fb2863b";
+    sha256 = "sha256-AhxCWWqCjlOroqs4pOSZTWoIQT8a1l/D2Rxuw1XUoU8=";
+  };
+
+  propagatedBuildInputs = [
+    python-dateutil
+    ephem
+    pytz
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "lunarcalendar" ];
+
+  meta = with lib; {
+    homepage = "https://github.com/wolfhong/LunarCalendar";
+    description = "A Lunar-Solar Converter, containing a number of lunar and solar festivals in China.";
```

```suggestion
    description = "A Lunar-Solar Converter, containing a number of lunar and solar festivals in China";
```
Please remove the trailing period.
https://nixos.org/manual/nixpkgs/stable/#var-meta-description